Back-End/middlewares/auth.js

```diff
@@ -0,0 +1,61 @@
+const jwt = require("jsonwebtoken");
+
+//checking if login or not
+const authentication = async (req, res, next) => {
```

Comment:
Use callbacks instead of `async` and `await` to have all the code made the same way